### PR TITLE
fix: resolve inconsistency between lock and toml

### PR DESCRIPTION
**Pull request by eloy-encord**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -78,7 +78,7 @@ statsmodels = "^0.13.5"
 umap-learn = "^0.5.3"
 streamlit-plotly-events = "^0.0.6"
 llvmlite = "^0.39.1"                                     # Pinning, as lower versions conflict with other libs
-encord-active-components = "^0.0.22"
+encord-active-components = "0.0.22"
```
Comment (james-encord) [inline pyproject.toml:81]: https://python-poetry.org/docs/dependency-specification/

use "==0.0.22" for strict exact

**Review by james-encord — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (eloy-encord) [reply pyproject.toml:81]: that's when using the command `add`

Comment (eloy-encord) [reply pyproject.toml:81]: not what appears in the toml

Comment (eloy-encord) [reply pyproject.toml:81]: btw, you shouldn't make modifications to the toml directly, just use the poetry CLI commands which guarantee that the state is preserved. Direct modification is what got us to this fix.

**Pull request merged**